plugins/system-update/click/apiclient_impl.cpp

```diff
@@ -78,33 +83,26 @@ void ApiClientImpl::requestMetadata(const QUrl &url,
 
 void ApiClientImpl::requestUpdatesMetadata(const QStringList &packages)
 {
-    QUrl url(Helpers::clickMetadataUrl());
-    QUrlQuery query(url);
-
-    QJsonObject body;
-
-    QStringList frameworks = Helpers::getAvailableFrameworks();
-    body.insert("frameworks", QJsonArray::fromStringList(frameworks));
-
-    body.insert("architecture",
-                QString::fromStdString(Helpers::getArchitecture()));
-
-    body.insert("apps", QJsonArray::fromStringList(packages));
-
-    body.insert("channel", Helpers::getSystemCodename());
-
-    QJsonDocument doc(body);
-    QByteArray content = doc.toJson();
-
-    QUrl u(url);
-    u.setQuery(query);
-    QNetworkRequest request;
-    request.setHeader(QNetworkRequest::ContentTypeHeader, "application/json");
-    request.setUrl(u);
-    request.setOriginatingObject(this);
-    request.setAttribute(QNetworkRequest::User, "metadata-request");
-
-    initializeReply(m_nam->post(request, content));
+    m_hasErrors = false;
+    foreach (const auto &package, packages) {
+        QString rawUrl(Helpers::clickMetadataUrl());
+        rawUrl.append(package);
+        QUrl url(rawUrl);
+        QUrlQuery query(url);
+
+        qWarning() << url;
```

Comment:
I suggest printing `u` a few lines below, so we also see the query parameters.

Reply:
Ups, was not meant to be included